Dockerfile

```diff
@@ -48,8 +49,7 @@ ENTRYPOINT ["sleep", "infinity"]
 FROM base AS kubectl
 
 RUN apt update && \
-      apt install -y --no-install-recommends  \
-      ca-certificates && \
+      apt install -y --no-install-recommends && \
```

Comment:
I think this is not needed anymore, if there was a update this is already done in the base image.